test/docs/dummy.yml

```diff
@@ -188,12 +188,15 @@ components:
       payload:
         $ref: "#/components/schemas/dummyInfo"
       examples:
-        - headers:
+        - #name: option1example
```

Comment:
It should be uncommented also another comments.

Reply:
omg!